decidim-comments/app/cells/decidim/comments/comment_cell.rb

```diff
@@ -33,8 +33,25 @@ def votes
         render :votes
       end
 
+      def perform_caching?
+        super && has_replies? == false
+      end
+
       private
 
+      def cache_hash
+        return @hash if defined?(@hash)
```

Comment:
Any reason for using an instance variable in only this cell's cache_hash method?

Reply:
If i remember correctly this is using a instance variable as the cache key is being used several times, and each one of those times the reported query is being executed.

Reply:
Ok, that makes sense!